apps/web/src/components/tx/FieldsGrid/index.tsx

```diff
@@ -19,7 +19,7 @@ const FieldsGrid = ({ title, children }: { title: string | ReactNode; children:
       ]}
     >
       <Grid item data-testid="tx-row-title" style={{ wordBreak: 'break-word' }} sx={gridSx}>
-        <Typography color="primary.light" variant="body1" component="span">
```

Comment:
This was updated as part of the native earn decoding to always have 16px of font-size on the confirmation screen: https://www.figma.com/design/K1EOvrgmK59lIPTM3WDHpm/Lending?node-id=5078-16132&m=dev
Are there exceptions where we want to keep the 14px and can we list them? cc @TanyaEfremova

Reply:
Thanks for clarifying. I agree for long data strings like that going with a smaller font makes more sense.